### DISCUSSION TITLE
收集有參考價值的資料

### DISCUSSION THREAD

**Discussion by scrum-1** [mdecd2023/football-apj1#2] [General]
https://github.com/jchoi2507/CoppeliaSim-Remote-API

https://github.com/Troxid/vrep-api-python

https://github.com/kanishkaganguly/VREPQuadcopter

https://github.com/HrushikeshBudhale/Serving_robot

https://github.com/hypagedev/coppelia_gui_robot_control_python

https://github.com/karadalex/vrep-raven2

https://github.com/chauby/BipedalWalkingRobots

https://github.com/sagarchotalia/Pick-and-Place-Robot-Eklavya

https://github.com/fakhrip/Robotics-Research

**Comment by mdecycu** [comment]
https://github.com/robocomp/V-REP

**Reply by scrum-1** [reply]
將倉儲 fork 到 https://github.com/scrum-1/V-REP, 其中的 https://github.com/scrum-1/V-REP/blob/master/doc/V-REP_API.md 說明如何利用 Python remote API 控制 BubbleRob. 假如再加上對 CoppeliaSim 套件根目錄中 remoteApiConnections.txt 與下拉式功能表 Modules -> Connectivity -> [Visualization stream](https://github.com/CoppeliaRobotics/threejsFrontend) 的了解, 應該可以讓多人直接透過瀏覽器與各自電腦的 Python remote API 程式控制 BubbleRob.

**Reply by scrum-1** [reply]
remoteApiConnections.txt 的原始內容:
```
// This file defines all the continuous remote API server services (started at remote API plugin initialization, i.e. CoppeliaSim start-up)
//
// Each remote API server service requires following 3 entries:
//
// portIndex@_port = xxxx               // where xxxx is the desired port number (below 19997 are preferred for server services starting at CoppeliaSim start-up)
// portIndex@_debug = xxxx              // where xxxx is true or false
// portIndex@_syncSimTrigger = xxxx     // where xxxx is true or false. When true, then the service will be pre-enabled for synchronous operation.
//
// In above strings, @ can be any number starting with 1. If more than one server service is required, then numbers need to be consecutive and starting with 1

// Let's start a continuous remote API server service on port 19997:
portIndex1_port             = 19997
portIndex1_debug            = false
portIndex1_syncSimTrigger   = true
```

**Reply by scrum-1** [reply]
假如要使用更新的 bluezero 控制 BubbleRob: https://github.com/scrum-1/coppeliasim_pythonic_api
